plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -996,6 +999,61 @@ public void testAllowColumnDefaults()
         assertQueryFails("INSERT INTO allow_column_defaults (a) VALUES (2)", "\\QUnsupported writer features: [allowColumnDefaults]");
     }
 
```

Comment:
Can we also have a product test that table with deletion vector created by Trino is readable by DB/Spark ?